src/main/java/sirius/tagliatelle/macros/FormatTimeMacro.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.tagliatelle.macros;
+
+import sirius.kernel.commons.Value;
+import sirius.kernel.di.std.Register;
+import sirius.kernel.nls.NLS;
+import sirius.tagliatelle.Tagliatelle;
+import sirius.tagliatelle.expression.Expression;
+import sirius.tagliatelle.rendering.LocalRenderContext;
+
+import javax.annotation.Nonnull;
+import java.time.LocalTime;
+import java.time.temporal.TemporalAccessor;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Represents <tt>formatTime(…)</tt>.
+ *
+ * The macro supports the datatypes {@code long} for epoch milliseconds, {@link Date} for legacy applications, and
+ * {@link TemporalAccessor}. The invocation is forwarded to {@link NLS#toUserString(Object)} via
+ * {@link Value#asLocalTime(LocalTime)}.
+ */
+@Register
+public class FormatTimeMacro implements Macro {
```

Comment:
can't we subclass FormatDateMacro, as verifyArguments is same
-> or extract superclass?